vm/src/stdlib/os.rs

```diff
@@ -1127,6 +1127,95 @@ pub(super) mod _os {
         OutputMode::String.process_path(curdir_inner(vm)?, vm)
     }
 
+    #[cfg(unix)]
+    #[pyfunction]
+    fn register_at_fork(kwargs: crate::function::KwArgs, vm: &VirtualMachine) -> PyResult<()> {
+        let mut match_found = false; // better way to handle this?
+        for (key, value) in kwargs.into_iter() {
```

Comment:
no error when key matches none of 3 keys?

Reply:
For this reason I was using `let mut match_found = false` hack but it is not needed if I use `#[derive(FromArgs)]`

Reply:
If no errors raised for wrong keys, adding a useless KwArgs will be helpful

e.g.
```rust
fn register_at_fork(args: RegisterAtForkArgs, _ignored: Kwargs, vm: ..) -> ...
```

Reply:
I used this but I am getting
```
TypeError: Unexpected keyword argument
```